app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -122,7 +131,11 @@ class PatientRepository @Inject constructor(
             val newOrUpdatedAddresses = payloads.map { it.address.toDatabaseModel() }
             database.addressDao().save(newOrUpdatedAddresses)
 
-            val newOrUpdatedPatients = payloads.map { it.toDatabaseModel(newStatus = DONE) }
+            val newOrUpdatedPatients = payloads.map {
+              it.toDatabaseModel(newStatus = DONE)
+            }.map {
+              it.copy(searchableName = strip(it.fullName))
```

Comment:
We should default searchableName to it value in the model itself.

Reply:
Do you mean make it an optional parameter?

Reply:
Resolved on a call